caluma/caluma_workflow/tests/test_work_item.py

```diff
@@ -499,7 +499,7 @@ def test_complete_work_item_with_next_multiple_instance_task(
         is_multiple_instance=True, address_groups=["group1", "group2", "group3"]
     )
     task_flow = task_flow_factory(task=task, workflow=workflow)
-    task_flow.flow.next = f"['{task_next.slug}']|task"
+    task_flow.flow.next = f"['{task_next.slug}']|tasks"
```

Comment:
Does this indicate a breaking change? Maybe it's too early in the morning, but to my understanding, this should never have worked :thinking:

Reply:
This task flow couldn't exist since we validate this on creation. The test was wrong and it shouldn't have worked. However, since it would have failed in the validation, I don't recognize this as breaking..

Reply:
Thanks for clarifying.